methods_and_flow_control/string_practice.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+
+my_str = 'this is a string to practice with'
+
+puts my_str
+puts my_str.capitalize
+puts my_str.upcase
+puts my_str.gsub('string',"'string'").capitalize
+puts "The string '#{my_str }' has #{my_str.length} characters"
+puts my_str.reverse
+puts (my_str.slice(20..28)*3).chop()+"!"
```

Comment:
The ()'s are optional.  Doesn't Rubocop complain about the lack of spaces around the `+`?